tests/Domain/FundTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace MatchBot\Tests\Domain;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use MatchBot\Domain\CampaignFunding;
+use MatchBot\Domain\ChampionFund;
+use MatchBot\Domain\Money;
+use MatchBot\Domain\Salesforce18Id;
+use MatchBot\Tests\TestCase;
+
+class FundTest extends TestCase
+{
+    private ChampionFund $fund;
+
+    public function setUp(): void
+    {
+        $this->fund = new ChampionFund('GBP', 'Testfund', Salesforce18Id::of('sfFundId4567890abc'));
+
+        $this->fund->addCampaignFunding(new CampaignFunding(
+            fund: $this->fund,
+            amount: '123.45',
+            amountAvailable: '100.00',
+            allocationOrder: 100,
+        ));
+    }
+
+    public function testGetAmounts(): void
```

Comment:
probably worth testing this a bit more thoroughly, e.g. test cases with zero, one and many amounts to sum. Might be a good fit for a data-driven test.